### DISCUSSION TITLE
background service GeLocation

### DISCUSSION THREAD

**Discussion by Ahmadmansoor** [dotnet/maui#9346] [Q&A]
are we still unable to do background services ?
if not, how are we able to build an advanced app?

**Comment by aritchie** [comment]
This isn't something MAUI or any other mobile ecosystem offers out of the box.  You have to go out and dev this yourself or use a 3rd party ecosystem package.

For MAUI, you can use https://github.com/shinyorg/shiny which has alpha support for MAUI right now
